cmd/notary-server/config.go

```diff
@@ -61,25 +63,43 @@ func grpcTLS(configuration *viper.Viper) (*tls.Config, error) {
 }
 
 // parses the configuration and returns a backing store for the TUF files
-func getStore(configuration *viper.Viper, allowedBackends []string, hRegister healthRegister) (
+func getStore(configuration *viper.Viper, hRegister healthRegister) (
 	storage.MetaStore, error) {
+	var store storage.MetaStore
+	backend := configuration.GetString("storage.backend")
+	logrus.Infof("Using %s backend", backend)
 
-	storeConfig, err := utils.ParseStorage(configuration, allowedBackends)
-	if err != nil {
-		return nil, err
-	}
-	logrus.Infof("Using %s backend", storeConfig.Backend)
-
-	if storeConfig.Backend == utils.MemoryBackend {
+	switch backend {
+	case notary.MemoryBackend:
 		return storage.NewMemStorage(), nil
+	case notary.MySQLBackend, notary.SQLiteBackend:
+		storeConfig, err := utils.ParseSQLStorage(configuration)
```

Comment:
Technically it's not parse MySQL storage. It parses any storage that's compatible with the ORM, which includes MySQL, SQLite, PostgreSQL, and I think MSSQL. We may also change ORM if anyone ever asks for Oracle support (easier to support one ORM for the whole SQL family).